platform/broadcom/sonic-platform-modules-ruijie/common/script/fancontrol.py

```diff
@@ -0,0 +1,837 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+import sys
+import click
+import os
+import commands
+import time
+from  ruijieutil import * 
```

Comment:
lgtm [py/non-iterable-in-for-loop
Not interesting about this!

Reply:
Rather than `from  ruijieutil import *`, you can instead `import ruijieutil` and reference everything as `ruijieutil.<name>`. This will fix the LGTM alert.